book/tutorials/uavsar/3_interferometric_swe_inversion.ipynb

```diff
@@ -174,7 +174,8 @@
     "ax.set_ylim(38.75, 39.3)\n",
     "\n",
     "# Add background map\n",
-    "cx.add_basemap(ax, crs=img.rio.crs, source = cx.providers.Stamen.TerrainLabels)\n",
+    "# Unfortunately, the below line does not render as of 20240301\n",
+    "# cx.add_basemap(ax, crs=img.rio.crs, source = cx.providers.Stamen.TerrainLabels)\n",
```

Comment:
Sad to see this not working with today's run anymore. Luckily it is not breaking essential parts of the tutorial.

Reply:
Yes, sadly since 10/2023 Stamen-provided map tiles require an API key (https://xyzservices.readthedocs.io/en/stable/changelog.html#xyzservices-2023-10-1-october-26-2023)

Reply:
Thanks for sharing and good to know